pkgs/applications/altcoins/ledger-live-desktop/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+ stdenv,
+ appimage-run,
+ fetchurl,
+ runtimeShell,
+}:
+
+stdenv.mkDerivation rec {
+  github-user = "LedgerHQ";
+  pname = "ledger-live-desktop";
+  version = "1.9.1";
+
+  src = fetchurl {
+    url = "https://github.com/${github-user}/${pname}/releases/download/v${version}/${pname}-${version}-linux-x86_64.AppImage";
+    sha256 = "0b919ilvv3zi17fnzngnnmg28dxqlq0dvj3fdvb50kh3ibrhdcfz";
+  };
+
+  buildInputs = [ appimage-run];
+
+  unpackPhase = ":";
+
+  installPhase = ''
```

Comment:
I'm not sure on this but would it be a good idea to run the preInstall and postInstall hooks so that people can override them